src/app/ui/autocomplete/Autocomplete.tsx

```diff
@@ -46,8 +48,12 @@ class Autocomplete extends PureComponent<AutocompleteProps> {
                     getMenuProps,
                     getItemProps,
                     highlightedIndex,
-                }) => (
+                }) => {
+                    const downshiftInputProps = getInputProps();
+
+                    return (
                     <div>
+                        <span id={ downshiftInputProps['aria-labelledby'] } style={ {width: 100, height: 100, position: 'absolute', top: ' -2000px ', left: 0} }>{ labelContent }</span>
```

Comment:
I think a better approach might be to pass `labelId` to `Downshift` component so that it doesn't have auto-generate a label ID that points to a non-existent element (https://github.com/downshift-js/downshift#labelid). And then you can point to the actual label that's visible to the users instead of having to create an invisible one?

Reply:
Thanks for the feedback. I  have changed the code and tested it on Launchbay.  Also changed the `for` value of the `<label>` to the correct `id` of the `<input>`.